CONTRIBUTING.md

```diff
@@ -35,7 +35,8 @@ We welcome many different types of contributions and not all of them need a Pull
 This is a rough outline of what a contributor's workflow looks like:
 
 * Make a fork of the repository within your GitHub account
-* Create a topic branch in your fork from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
+   * Currently we accept contribution from forks only for examples and documentation changes until PR 854 is fixed. Until then please request write priveleges and create a branch in the main repo.
+* Create a topic branch from where you want to base your work - name your branch according to our naming convention - person/<github-username>/<feature-name>
```

Comment:
Move `Create a topic branch` as primary way of contributing, above the fork statement.

Reply:
Since we can accept some PRs using forks , Creating a fork for that case it is still a first step.